apps/backend-e2e/src/e2e/user.cy.ts

```diff
@@ -0,0 +1,472 @@
+      body: {
+        email: mail,
+        password: faker.internet.password(),
+        role: Roles.EDITOR,
+        active: true,
+      },
+      auth: {
+        bearer: adminToken,
+      },
+      failOnStatusCode: false,
+    }).then((response) => {
+      expect(response.status).to.eq(200);
+      expect(response.body.email).to.eq(mail);
+      expect(response.body.role).to.eq(Roles.EDITOR);
+      expect(response.body.active).to.eq(true);
+      expect(response.body.id).to.eq(String(createdEditorId));
+    });
+  });
+  it('Should return data user when an editor tries to update his/her own data.', () => {
+    const mail = faker.internet.email();
+    cy.request({
+      method: 'PUT',
+      url: `/api/users/${editorId}`,
+      body: {
+        email: mail,
+        password: faker.internet.password(),
+        role: Roles.EDITOR,
+        active: true,
+      },
+      auth: {
+        bearer: editorToken,
+      },
+      failOnStatusCode: false,
+    }).then((response) => {
+      expect(response.status).to.eq(200);
+      expect(response.body.email).to.eq(mail);
+      expect(response.body.role).to.eq(Roles.EDITOR);
+      expect(response.body.active).to.eq(true);
+      expect(response.body.id).to.eq(String(editorId));
+    });
+  });
+  it('Should fail when an editor tries to modify another user.', () => {
+    const mail = faker.internet.email();
+    cy.request({
+      method: 'PUT',
+      url: `/api/users/${createdEditorId}`,
+      body: {
+        email: mail,
+        password: faker.internet.password(),
+        role: Roles.EDITOR,
+        active: true,
+      },
+      auth: {
+        bearer: editorToken,
+      },
+      failOnStatusCode: false,
+    }).then((response) => {
+      expect(response.status).to.eq(403);
+    });
+  });
+  it('Should fail when updating user with already used email.', () => {
+    cy.request({
+      method: 'PUT',
+      url: `/api/users/${createdEditorId}`,
+      body: {
+        email: 'admin@example.org',
+        password: faker.internet.password(),
+        role: Roles.EDITOR,
+        active: true,
+      },
+      auth: {
+        bearer: adminToken,
+      },
+      failOnStatusCode: false,
+    }).then((response) => {
+      expect(response.status).to.eq(400);
+    });
+  });
+  // DELETE
+  it('Should fail when an editor tries to delete any user.', () => {
+    cy.request({
+      method: 'DELETE',
+      url: `/api/users/${createdEditorId}`,
+      auth: {
+        bearer: editorToken,
+      },
+      failOnStatusCode: false,
+    }).then((response) => {
+      expect(response.status).to.eq(403);
+    });
+  });
+  it('Should return HTML code 204 when admin suppresses a user.', () => {
+    cy.request({
+      method: 'DELETE',
+      url: `/api/users/${createdEditorId}`,
+      auth: {
+        bearer: adminToken,
+      },
+      failOnStatusCode: false,
+    }).then((response) => {
+      expect(response.status).to.eq(204);
+    });
+  });
+});
```

Comment:
Since this file is 500 lines long, could we separate it into few smaller files ?